### ISSUE TITLE
[Enhancement] Additional Group placeholder

### ISSUE THREAD

**Issue by redadykai** [promcteam/proskillapi#400]
**Is your feature request related to a problem? Please describe.**
i would like to have a placeholder that will return values from group class like  in player placeholder

**Describe the solution you'd like**
For ex. 

This will only return the value from player and if that player is on the same account it will not display the info from another account
- %sapi_player_<groupname>_mainclass%
- %sapi_player_<groupname>_level%  

i would like to see something like this as well 
%sapi_group_<groupname>_mainclass%
%sapi_group_<groupname>_level% 

so that it will return the value from group which will give way displaying the information of that character even if that said character is not on the same account.

basically want it want to do is to use the placeholders in a different gui. 

**a video for better understanding**
[2022-10-25 23-33-57-1.webm](https://user-images.githubusercontent.com/62630889/197818249-b8ceb154-5d94-4ed6-8d8f-e75a03d556e7.webm)

**Comment by redadykai** [comment]
1 week no response.

**Comment by goflishMC** [comment]
We don't reply to every requested new feature, but I do read them all.

**Comment by DrHeretic** [comment]
![image](https://user-images.githubusercontent.com/88398845/199424018-8d65a860-7a7d-4123-b833-d73a29d71470.png)
![image](https://user-images.githubusercontent.com/88398845/199424044-3303060f-9088-4ea7-a76b-175bac79709b.png)

It is already there

**Comment by redadykai** [comment]
its not.

**Comment by redadykai** [comment]
group placeholders are meant to call the values if you are not using the current character, via accounts, so technically. 
we need something like,

player class
player level

to be viewed via group placeholders, this can be used in custom GUIs, i think that is what the previous developer is trying to do with that prior you guys taking over.

**Comment by lordsrubka5** [comment]
its poorly explained, what he wants is a placeholder to parse information from /class list
```
%sapi_account_1_class%
%sapi_account_1_level%
2,3,4,5,6,7...
```

**Comment by redadykai** [comment]
> 

yeah, sorry english is not my first language, but yeah you're right so basically having account 2 visible in gui using placeholder while player is on account 1
